lib/trino-collect/src/main/java/io/trino/collect/cache/EmptyCache.java

```diff
@@ -116,7 +116,7 @@ public ConcurrentMap<K, V> asMap()
             public V putIfAbsent(K key, V value)
             {
                 // Cache, even if configured to evict everything immediately, should allow writes.
-                return value;
+                return null;
```

Comment:
the above comment looks weird now

Reply:
fixed myself

Reply:
Thanks

Reply:
But you forced pushed.
Would you consider
```
// putIfAbsent returns the previous value
```

Reply:
Sorry about that 🤦 

Notice that the put method has the old comment still, I find old comment still useful. It is more about empty cache, not the put method. I will leave both ones.